ruby/vampires.rb

```diff
@@ -28,6 +35,20 @@
 puts "Would you like to enroll in the company's health insurance?"

```

Comment:
For line 18.

When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`